docs/fundamentals/flow/gateway-args.md

```diff
@@ -38,6 +38,7 @@
 | `graph_description` | Routing graph for the gateway | `string` | `{}` |
 | `graph_conditions` | Dictionary stating which filtering conditions each Executor in the graph requires to receive Documents. | `string` | `{}` |
 | `deployments_addresses` | dictionary JSON with the input addresses of each Deployment | `string` | `{}` |
+| `deployments_metadata` | dictionary JSON with the request metadata for each Deployment | `string` | `{}` |
 | `deployments_disable_reduce` | list JSON disabling the built-in merging mechanism for each Deployment listed | `string` | `[]` |
```

Comment:
- Capitalize first letter of each description
- Should it be "JSON dictionary", not "dictionary JSON"? I'm confused - is it in `dict` format or a JSON string?

Reply:
It is a JSON string.

Reply:
So, I will go with "JSON dictionary"

Reply:
cool cool